### PR TITLE
Add more elements

### PR DESCRIPTION
**Pull request by yuanyan**
1.  Add block element: h1-h6 main header aside...
2. Add inline element: small i code samp...  close https://github.com/openkraken/kraken/issues/110
3. Default unknow element display change to inline

### CODE REVIEW

File: integration_tests/specs/dom/elements/br.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+fdescribe('br-element', () => {
```
Comment (wssgcg1213) [inline integration_tests/specs/dom/elements/br.ts]: No `f`

File: kraken/macos/libkraken_jsc.dylib [hunk 2]
```diff
@@ -1 +1 @@
-../../bridge/build/macos/lib/x86_64/libkraken_jsc.dylib
+/Users/yuanyan/Kraken/kraken/kraken/macos/../../bridge/build/macos/lib/x86_64/libkraken_jsc.dylib
```
Comment (wssgcg1213) [inline kraken/macos/libkraken_jsc.dylib:1]: 这个文件不要提交上来

**Review by wssgcg1213 — CHANGES_REQUESTED** [review]

File: kraken/lib/src/dom/elements/semantics_text.dart [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+const Map<String, dynamic> _sDefaultStyle = {
+  TEXT_DECORATION: LINE_THROUGH
+};
+
+const Map<String, dynamic> _smallDefaultStyle = {
+  FONT_SIZE: SMALLER
+};
+
+const Map<String, dynamic> _codeDefaultStyle = {
+  FONT_FAMILY: 'monospace'
```
Comment (answershuto) [inline kraken/lib/src/dom/elements/semantics_text.dart:51]: 是否应该定义成常量？
const String MONOSPACE = 'monospace';

File: kraken/lib/src/dom/elements/object.dart [hunk 4]
```diff
@@ -65,11 +66,11 @@ class ObjectElement extends Element implements ObjectElementHost {
...
-  @override
-  void setStyle(String key, value) {
-    super.setStyle(key, value);
-    _objectElementClient?.setStyle(key, value);
-  }
+  // @override
+  // void setStyle(String key, value) {
+  //   super.setStyle(key, value);
+  //   _objectElementClient?.setStyle(key, value);
+  // }
```
Comment (answershuto) [inline kraken/lib/src/dom/elements/object.dart:73]: 删除注释

File: kraken/lib/src/dom/elements/semantics_text.dart (hunk 3 above)
Comment (yuanyan) [reply kraken/lib/src/dom/elements/semantics_text.dart:51]: 因为字体，字体无法穷举

**Review by wssgcg1213 — APPROVED** [review]

**Review by answershuto — APPROVED** [review]

**Review by andycall — APPROVED** [review]

**Pull request merged**